container_build.sh

```diff
@@ -70,7 +70,14 @@ build() {
 }
 
 determine_platform() {
-  local platform="linux/amd64"
+  case $conman_bin in
+    podman)
+      local platform="$(podman info --format '{{ .Version.OsArch }}' 2>/dev/null)"
+      ;;
```

Comment:
This is fine, but why not continue to use "uname -m" like all the other CPU architectures, they you don't have to worry about podman/docker differences at all, it's easier to test and maintain.

Reply:
Unfortunately container world is using the Go arch strings that are not compatible with uname -m.